acceptance/tests/server_list_setting.rb

```diff
@@ -0,0 +1,58 @@
+test_name "Priority of server_list setting over server setting" do
+  master_port = 8140
+
+  master_opts = {
```

Comment:
I think this may be unnecessary, as masterport is 8140 by default. You can do the following instead:

```
with_puppet_running_on(master, {}) do
...
end
```